app/src/main/java/net/skyscanner/backpack/demo/components/SingleSelectChipGroupComponent.kt

```diff
@@ -0,0 +1,24 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.demo.components
+
+import net.skyscanner.backpack.meta.ComponentMarker
+
+@ComponentMarker("SingleSelectChipGroup")
+annotation class SingleSelectChipGroupComponent
```

Comment:
While we have a distinction between single & multi select the main component is still `ChipGroup` - we should have both single & multi select components in the same package & story. You can rename this marker annotation to `ChipGroupComponent` (and the name in the annotation to `Chip Group` as that's the readable name), as well as the story and then rename the package of the component to `chipgroup`